### PR TITLE
DOM - Added documentation on the new ITrackBase fields + Added examples on how to alter values on a DomInstance

### PR DESCRIPTION
**Pull request by Skyline-ThomasGH**

### CODE REVIEW

File: user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+- Add/update a value to/of a DomInstance.
+- Get a value from a DomInstance.
+
+These two methods have a variant for a single value and a list value.
+
+There are also two versions available, one where you can pass the full `SectionDefinition` and `FieldDescriptor` objects, and one where you only need to pass the IDs of these objects. The latter has the advantage that you do not need to retrieve these full objects from server, which could improve your script's performance. Do note, however, that the former will automatically [stitch](xref:DomHelper_class#stitching) these full objects into the DomInstance automatically. (Not really necessary in most cases)
+
+> [!NOTE]
+> The version of these extension methods that accept the IDs are available since DataMiner version 10.3.2.
```
Comment (VanRobaysWouter) [inline user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md:45]: Also add the main release version where it will be available in

File: user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+
+## Manual altering Sections and FieldValues
+
+It is also possible to manually add `Sections` and `FieldValues` to the `DomInstance`. This could be required when you, for example, want to add multiple `Sections` for the same `SectionDefinition`.
+
+### Adding a new Section & FieldValue
+
+```csharp
+// Define the ID of the DomDefinition, the SectionDefinition and the ID for one of its FieldDescriptors.
```
Comment (EdithLansens) [inline user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md:121]: @Skyline-ThomasGH Are these code blocks examples of e.g. adding a new `Section` and `FieldValue`? If so, I will add an "###Examples" subtitle, similar to the previous section on simple extension methods.

Comment (Skyline-ThomasGH) [reply user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md:121]: Yes indeed. Thanks.

File: user-guide/Advanced_Modules/DOM/DOM_ModuleSettings.md [hunk 3]
```diff
@@ -70,3 +70,5 @@ The current settings structure (with regards to `DomManager`) is as follows:
 - To include the module settings in a [custom backup](xref:Backing_up_a_DataMiner_Agent_in_DataMiner_Cube#configuring-the-dataminer-backups) in Cube, select *Create a backup of the database* > *Include module specific configuration data*.
 
 - There are no license checks when module settings are added.
+
+ - [The ITrackBase properties](xref:DOM_objects#itrackbase-properties) are also available on this type.
```
Comment (EdithLansens) [inline user-guide/Advanced_Modules/DOM/DOM_ModuleSettings.md:74]: @Skyline-ThomasGH Can I change this sentence to:
"The *ModuleSettings* object also has the *ITrackBase* properties."?
This way of wording is slightly more user-friendly, but I want to make sure it does not change the meaning.

File: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
 
 ## CustomSectionDefinition properties
 
-The table below lists the properties of the `CustomSectionDefinition` object. (The base `SectionDefinition` object only exposes its ID.) The table also indicates whether a property can be used for filtering using the `SectionDefinitionExposers`.
+The table below lists the properties of the `CustomSectionDefinition` object. (The base `SectionDefinition` object only exposes its ID.) The table also indicates whether a property can be used for filtering using the `SectionDefinitionExposers`. Note that [the ITrackBase properties](xref:DOM_objects#itrackbase-properties) are also available on this type.
```
Comment (EdithLansens) [inline user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md:111]: @Skyline-ThomasGH Can I change this to:
"The *CustomSectionDefinition*  object also has the *ITrackBase* properties."?

File: user-guide/Advanced_Modules/DOM/DOM_objects/DomTemplate.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ A `DomTemplate` object stores a `DomInstance` with some pre-defined data, so tha
 
 ## Properties
 
-The table below lists the properties of the `DomTemplate` object. It also indicates whether a property can be used for filtering using the `DomTemplateExposers`.
+The table below lists the properties of the `DomTemplate` object. It also indicates whether a property can be used for filtering using the `DomTemplateExposers`. Note that [the ITrackBase properties](../DOM_objects.md#itrackbase-properties) are also available on this type.
```
[the remark this answers is not in the document]
Comment (Skyline-ThomasGH) [reply user-guide/Advanced_Modules/DOM/DOM_objects/DomTemplate.md:11]: All the recommendations are fine for me. Thanks.

File: user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+
+// Retrieving a list value.
+var fieldValue = section.GetFieldValueById(fieldDescriptorId);
+var valueWrapper = fieldValue.Value as ListValueWrapper<string>;
+List<string> actualValue = valueWrapper.Values;
+```
+
+## Notes
+
+- Make sure that the value is of the same type as was defined in the associated `FieldDescriptor`. We recommend that you use an IDE that allows you to easily see the type by hovering over it. Deceiving situations might occur, e.g. an 'int' is assigned but the `FieldDescriptor` expects type 'long' (Use '236L' instead of '236' in this case).
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md:274]: Could you use double quotes instead of single quotes here? To match the Microsoft style, we should use double quotes everywhere except in titles (as markdown does not support them there).

File: user-guide/Advanced_Modules/DOM/DOM_objects.md [hunk 7]
```diff
@@ -35,3 +35,33 @@ For example, to create a system that stores time spent working on specific tasks
 - Field descriptor 4: *Name* = "Stop time" and *Type* = "DateTime"
 
 A DOM definition will then need to be created that contains a link to this section definition, so that employees can then register time by creating a new DOM instance linked to that DOM definition. The DOM instance will have a section linked to the section definition that contains a field value for each of the field descriptors.
+
+## ITrackBase Properties
+
+From DataMiner 10.3.2/10.4.0 onwards, all DOM objects (including the `ModuleSettings`) contain four properties that reflect who has created/updated the object, and at what time.
+
+- **LastModified** : The moment when the object was last modified, in UTC DateTime format.
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/DOM/DOM_objects.md:43]: Could you remove the space before the colons in this list? Usually, there's no space before a colon in English, so unless there's a very specific reason to have one after all, we should avoid this.

File: user-guide/Advanced_Modules/DOM/DOM_objects.md [hunk 8]
```diff
@@ -35,3 +35,33 @@ For example, to create a system that stores time spent working on specific tasks
...
+```
+
+It is also possible to use these fields to build filters when reading objects.
+
+```csharp
+var filter = DomInstanceExposers.CreatedBy.Equal("John Doe");
+```
+
+> [!NOTE]
+> The DOM objects created prior to DataMiner 10.3.2/10.4.0 will not have a value for these fields. However, after upgrading and updating an existing object once, the `LastModified` and `LastModifiedBy` fields will be filled in.
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/DOM/DOM_objects.md:67]: This second sentence contains a dangling participle. Could you rephrase as "However, after an existing object is upgraded and updated once, ..."?

**Review by MariekeGO — CHANGES_REQUESTED** [review]
@EdithLansens, could you look into implementing these last changes?

**Review by MariekeGO — APPROVED** [review]
